README.md

```diff
@@ -54,7 +54,12 @@ Contributions are always welcome! Please take a look at the [contribution guidel
 - [GroupMe Desktop Client](https://github.com/alexdillon/GroupMeClientAvalonia) - GroupMe Desktop Client Avalonia is an open-source, cross platform, modular client for GroupMe messenging.
 
 ## Tutorials
-
+- [Quickstart](https://avaloniaui.net/docs/quickstart/) - Info for quick start with Avalonia UI.
+- [AvaloniaUI Tutorial](https://avaloniaui.net/docs/tutorial/) - Official AvaloniaUI tutorial.
+- [Avalonia for WPF Developers](https://avaloniaui.net/docs/quickstart/from-wpf) - Quick start for WPF developers.
```

Comment:
Three of links are refer to official documentation. So it's a bit unclear why have you decided to split them up?

Reply:
Yes I understand  that there are links on same resource. But as for my opinion it's the most important part, and avalonia users needs short links to important parts of avalonia tutorial